cmd/tea/gen_id.go

```diff
@@ -128,6 +128,10 @@ func (i *{{ . }}) UnmarshalText(data []byte) error {
 func (i *{{ . }}) Scan(data any) error {
 	return scanUUID((*uuid.UUID)(i), "{{ . }}", data)
 }
+
+func (i *{{ .  }}) Equal(other *{{ . }}) bool {
```

Comment:
The `uuid.UUID` type is a byte array (`[16]byte`), so you can directly compare two UUIDs with the `==` operator. Example:
```
x := id.NewUser()
y := id.NewUser()
if x == y {
    fmt.Println("1")
} else {
    fmt.Println("0")
}
```
Could you please provide some justification (or examples of your usage) for having this method?